.prospector.yaml

```diff
@@ -0,0 +1,43 @@
+output-format: grouped
```

Comment:
Can we copy the PyLint setup from h-matchers instead of using Prospector? We're trying to move everything over to PyLint but this hasn't been done for the other apps yet.

* `.pylintrc`: https://github.com/hypothesis/h-matchers/blob/master/.pylintrc
* Separate pylintrc for tests: https://github.com/hypothesis/h-matchers/blob/master/tests/.pylintrc
* pydocstyle config file: https://github.com/hypothesis/h-matchers/blob/master/.pydocstyle
* tox installs pylint and pydocstyle: https://github.com/hypothesis/h-matchers/blob/2cca71bef1c6ec85979cf3404000bcdc8a4c84bb/tox.ini#L22-L23
* tox runs pylint and pydocstyle: https://github.com/hypothesis/h-matchers/blob/2cca71bef1c6ec85979cf3404000bcdc8a4c84bb/tox.ini#L40-L43
* No prospector

Reply:
I had a query on slack about this [here](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1575595245079900).

Reply:
Let's disable D213 as the matches PEP 257